client/wfnews-war/src/main/angular/src/app/components/admin-incident-form/admin-incident-form.component.scss

```diff
@@ -125,6 +125,24 @@
 			padding: .5rem 1rem;
 		}
 
+    .size-quarter {
+			flex: 0 0 25%;
+			box-sizing: border-box;
+			padding: .5rem 1rem;
```

Comment:
I usually add paddings in "px". we need to agree on either using rem or px in the future styling

Reply:
Well, if you prefer an "either or" approach we should only use rem, as it's not an absolute unit like px. However I would argue there are times when we'd benefit from using both